addons/account/models/account_move.py

```diff
@@ -287,6 +287,7 @@ def _get_default_invoice_incoterm(self):
         help="Auto-complete from a past bill.")
     invoice_source_email = fields.Char(string='Source Email', tracking=True)
     invoice_partner_display_name = fields.Char(compute='_compute_invoice_partner_display_info', store=True)
+    is_inactive_currency = fields.Boolean(compute="_compute_is_inactive_currency")
```

Comment:
add tooltip (help="Technical field used for blablaba")

Reply:
Why a field and not just a method? (field makes sense when used in views, ...)